packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -66,191 +70,194 @@ const getBackgroundColors = () => {
 
 const cfg = (options?: Options): Config => {
   const { disablePreflight = true } = options || {};
+  // make palette colors (foundation colors) defined as css vars and make components tokens inherit it
+  const theme = getTokens(cssVarsFoundation);
+  const componentTokens = getComponentLevelToken(theme);
```

Comment:
This is also hard to read. The function `getComponentLevelToken`, on its definition, names its parameter as `tokens`, but we're passing it a variable called `theme` (This was mentioned above). But then this function returns another function, it doesn't return a set of _componentTokens_ (which is what I infer when I read the variable name) that is then expected to be called, as we're doing. Please rename it. We can't call it again `getComponentLevelToken`, but then what do these two functions do differently?